README.md

```diff
@@ -245,25 +251,32 @@ Here are some examples of library usage: [gmock-win32-sample](https://github.com
 
 # Version history
 
+## Version 1.2.0 (** October 2023)
```

Comment:
It may be necessary to create a new version, as there have been significant changes.

Reply:
Feel joy when a new version appears. A lot of work has been done.

Reply:
Yep, thanks for contributing!